justfile

```diff
@@ -23,6 +23,18 @@ setup:
     just install-doppelganger
     just install-zombie-bite
 
+# ------------------------- INSTALLING DEPENDENCIES -------------
+
+install-doppelganger:
+    SKIP_WASM_BUILD=1 cargo install --git https://github.com/paritytech/doppelganger-wrapper --bin doppelganger \
+        --bin doppelganger-parachain \
+        --bin polkadot-execute-worker \
+        --bin polkadot-prepare-worker  \
+        --locked --root ${DOPPELGANGER_PATH}
```

Comment:
What is `DOPPELGANGER_PATH`? Some useful data the answer should contain: where is it defined, is it something the user needs to manually set, or is it automatically set by the `justfile`, etc.

Reply:
Nevermind, `cargo install --root` is what will give meaning to the env var.